tap_logic4/streams.py

```diff
@@ -224,6 +249,13 @@ class SupplierProductStream(Logic4Stream):
         )),      
     ).to_dict()
 
+    def prepare_request_payload(self, context, next_page_token):
+        return context["ProductId"]
```

Comment:
Is the payload meant to be just a string? Or should it be the full object? I think you want the full `context` dict to be returned here